addons/resource/tests/test_resource.py

```diff
@@ -358,38 +335,36 @@ def test_40_calendar_hours_scheduling(self):
 
         # old API: resource without leaves
```

Comment:
This test case is not for old API, but for deprecated method. Improve the comments for this and its succeeding test case.